Add 11.11 release notes (#241)
mondoohq/docs: releases/2024-07-02-mondoo-11.11-is-out.md

---
slug: mondoo-11.11-is-out/
title: Mondoo 11.11 is out!
description: Announcing the 11.11 release of Mondoo, with GitHub Enterprise support, automated IaC file discovery in code repositories, EU NIS2 Cybersecurity Directive Framework support, and more!
author: Tim Smith
author_title: Mondoo Core Team
author_url: https://github.com/tas50
image: /img/featured_img/release-feature.jpg
tags: [release, mondoo]
---

## 🥳 Mondoo 11.11 is out! This release includes GitHub Enterprise support, automated IaC file discovery in code repositories, EU NIS2 Cybersecurity Directive Framework support, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

### Continuous GitHub Enterprise scanning

> Mondoo's GitHub integration has expanded to support GitHub Enterprise as well. GitHub Enterprise users can now get all the Mondoo GitHub features for their private instances, including scans of their GitHub configuration, as well as automatic discovery of Terraform code and Kubernetes manifests in GitHub repositories.

Reviewer: ```suggestion
Mondoo's GitHub integration has expanded to support GitHub Enterprise as well. GitHub Enterprise users can now get all the Mondoo GitHub features for their private instances, including scans of their GitHub configuration and automatic discovery of Terraform code and Kubernetes manifests in GitHub repositories.
```

Check out the documentation to [learn more about integrating Mondoo Platform with GitHub Enterprise](https://mondoo.com/docs/platform/infra/saas/github/).

![GitHub Enterprise Configuration](/img/releases/2024-07-02-mondoo-11.11-is-out/gh-top.png)

### IaC file discovery in GitHub and GitLab

Mondoo now automatically discovers and scans Terraform code and Kubernetes manifests in GitHub and GitLab. If any static Terraform (HCL) or Kubernetes manifest files exist in a repo, Mondoo can detect and scan them for security misconfigurations. This option lets infrastructure developers responsible for writing and maintaining infrastructure as code detect security issues before they create problems in production.

Configure scanning in your existing GitHub and GitLab integrations.
![Automatic Discovery Options](/img/releases/2024-07-02-mondoo-11.11-is-out/automatic_discovery.png)

### NIS 2 Cybersecurity Directive compliance framework

Mondoo introduces a new NIS2 compliance framework for companies in the European Union that must comply with the NIS 2 cybersecurity directive.

![NIS2 Checks](/img/releases/2024-07-02-mondoo-11.11-is-out/nis-2.png)

### CIS Google Workspace benchmark policy

> the Google Workspace Foundations CIS Benchmark policy has been upgraded to match CIS benchmark version 1.2.

Author: ```suggestion
Secure your Google Workspace infrastructure with the new CIS Google Workspace Foundations Benchmark Level 1 and 2 policies including 58 important security checks.
```

---
slug: mondoo-11.11-is-out/
title: Mondoo 11.11 is out!
description: Announcing the 11.11 release of Mondoo, with GitHub Enterprise support, automated IaC file discovery in code repositories, EU NIS2 Cybersecurity Directive Framework support, and more!
author: Tim Smith
author_title: Mondoo Core Team
author_url: https://github.com/tas50
image: /img/featured_img/release-feature.jpg
tags: [release, mondoo]
---

## 🥳 Mondoo 11.11 is out! This release includes GitHub Enterprise support, automated IaC file discovery in code repositories, EU NIS2 Cybersecurity Directive Framework support, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

### Continuous GitHub Enterprise scanning

Mondoo's GitHub integration has expanded to support GitHub Enterprise as well. GitHub Enterprise users can now get all the Mondoo GitHub features for their private instances, including scans of their GitHub configuration, as well as automatic discovery of Terraform code and Kubernetes manifests in GitHub repositories.

> Check out the documentation to [learn more about integrationg Mondoo Platform with GitHub Enterprise](https://mondoo.com/docs/platform/infra/saas/github/).

Reviewer: ```suggestion
Check out the documentation to [learn more about integrating Mondoo Platform with GitHub Enterprise](https://mondoo.com/docs/platform/infra/saas/github/).
```

![GitHub Enterprise Configuration](/img/releases/2024-07-02-mondoo-11.11-is-out/gh-top.png)

### IaC file discovery in GitHub and GitLab

> Mondoo now automatically discovers and scans Terraform code and Kubernetes Manifests in GitHub and GitLab. Mondoo automatically scans static Terraform (HCL) and Kubernetes Manifests for security misconfigurations, enabling infrastructure developers responsible for writing and maintaining infrastructure-as-code for their organization to detect security issues before pushing changes to a remote repository.

Reviewer: ```suggestion
Mondoo now automatically discovers and scans Terraform code and Kubernetes manifests in GitHub and GitLab. If any static Terraform (HCL) or Kubernetes manifest files exist in a repo, Mondoo can detect and scan them for security misconfigurations. This option lets infrastructure developers responsible for writing and maintaining infrastructure as code detect security issues before they create problems in production.
```

Configure scanning in your existing GitHub and GitLab integrations.
![Automatic Discovery Options](/img/releases/2024-07-02-mondoo-11.11-is-out/automatic_discovery.png)

### NIS 2 Cybersecurity Directive compliance framework

Mondoo introduces a new NIS2 compliance framework for companies in the European Union that must comply with the NIS 2 cybersecurity directive.

![NIS2 Checks](/img/releases/2024-07-02-mondoo-11.11-is-out/nis-2.png)

### CIS Google Workspace benchmark policy

> the Google Workspace Foundations CIS Benchmark policy has been upgraded to match CIS benchmark version 1.2.

Reviewer: ```suggestion
The Google Workspace Foundations CIS Benchmark policy has been upgraded to match CIS benchmark version 1.2.
```

## 🧹 IMPROVEMENTS

### Oracle Linux Kernel CVE support

Mondoo now detects and reports vulnerabilities in the Oracle Unbreakable Enterprise Linux Kernel.

### DaemonSet-based Kubernetes integration scanning

> The Mondoo Kubernetes integration now supports running as either a DaemonSet, a Deployment or a Cronjob, depending on your needs.

Reviewer: ```suggestion
The Mondoo Kubernetes integration now supports running as either a DaemonSet, a Deployment or a CronJob, depending on your needs.
```

Kubernetes isn't always consistent in their docs but they tend to camel case it.

### Improved GitHub organization scanning scalability

Multiple improvements to the GitHub organization scanner now better ensure that Mondoo does not accidentally trigger GitHub's API rate limits.

### Improved Mondoo Terraform provider

The Terraform provider for Mondoo has several changes:

> - The Terraform provider has full access to the asset list of a Mondoo space and can leverage those assets in terraform hcl

Reviewer: ```suggestion
- The Terraform provider has full access to the asset list of a Mondoo space and can leverage those assets in Terraform HCL.
```

> - The Terraform provider now has access to the full list of active policies via a data source

Reviewer: ```suggestion
- The Terraform provider now has access to the full list of active policies via a data source.
```

> - Use Terraform to enable compliance frameworks and even upload custom frameworks

Reviewer: ```suggestion
- You can now use Terraform to enable Mondoo compliance frameworks and even upload custom frameworks.
```

### Resource updates

#### aws.rds.dbinstance

- New `subnets` field.

#### azure.subscription.aksService.cluster

- New `apiServerAccessProfile` field.

#### github.branch

- New `headCommitSha` field.

#### github.packages

- New resource to fetch information on packages for a repository.

## 🐛 BUG FIXES AND UPDATES

- Add vendor source links to VMware and Visual Studio Code security advisories.
- Add security advisories for openSUSE Tumbleweed.
- Fix scans failing to cancel in the AWS serverless integrations.
- Fix sort ordering of Blast Radius in tables.
- Fix display of the empty credentials box on the Generate Long-Lived Credentials page.
- Don't reject `.in` domains in the host integration setup page.
- Fix the settings link in the navigation bar sometimes disappearing.
- Improve the reliability of EBS volume scanning in AWS.
- Improve reliability of VM scans in the Azure integration.
- Update Fedora and AlmaLinux EOL dates to match the latest vendor announcements.

> - Update the EOL warning date to be when the date is 6 months out not 3.

Reviewer: ```suggestion
- Update the EOL warning date to be when the date is 6 months out instead of 3.
```

> - Fix an error displays checks for some policies in the console.

Reviewer: ```suggestion
- Fix an error displaying checks for some policies in the console.
```

- Fix a failure fetching the `attachedPolicies` data in the `aws.iam.users` resource.

> - Discovered AWS ECR images now include a runtime value of "aws-ecr".

Reviewer: ```suggestion
- Give discovered AWS ECR images a runtime value of "aws-ecr".
```

- Allow sorting compliance control checks in the console.

> - Add additional checks to the CIS AKS

Reviewer: ```suggestion
- Add new checks to the CIS AKS benchmark policy.
```

- Improve the reliability of CIS `Ensure default user umask is configured` and `Ensure default user umask is 027 or more restrictive` checks.
- Improve reliability of some Windows registry-based CIS checks.
- Simplify many Linux checks to improve result output.

> - Reduce screen flicking when filtering checks on asset pages.

Reviewer: ```suggestion
- Reduce screen flickering when filtering checks on asset pages.
```
